instruction-padding/program/src/lib.rs

```diff
@@ -2,5 +2,4 @@ mod entrypoint;
 pub mod instruction;
 pub mod processor;
 
-pub use solana_program;
```

Comment:
Breaking change, right?

Reply:
Ah oops! I thought I pushed up the latest change, which re-exports the new solana crates. But yes, this is otherwise breaking

Reply:
I guess if it's a breaking change anyway, do we want to still have the re-exports? Or should we just do away with them?

Reply:
It can't hurt in case someone needs to access it